rootston/output.c

```diff
@@ -17,13 +17,30 @@ static inline int64_t timespec_to_msec(const struct timespec *a) {
 	return (int64_t)a->tv_sec * 1000 + a->tv_nsec / 1000000;
 }
 
+/**
+ * Rotate a child's position relative to a parent. The parent size is (pw, ph),
+ * the child position is (*sx, *sy) and its size is (sw, sh).
+ */
+static void rotate_child_position(double *sx, double *sy, double sw, double sh,
+		double pw, double ph, float rotation) {
```

Comment:
Out args generally go last.

Reply:
So this is in _and_ out. Not sure what to do about it. Maybe make it in-only, and add another pair of out arguments?

Reply:
This is an internal rootston function, I don't care